src/Http/Middleware/HandleErrors.php

```diff
@@ -52,7 +52,7 @@ public function __invoke($error, Request $request, Response $response, callable
             $status = $errorCode;
         }
 
-        if ($this->debug && $errorCode !== 404) {
+        if ($this->debug && ! in_array($errorCode, [403, 404])) {
```

Comment:
Hmm I don't like this change... if debug mode is on, it's handy to have Whoops to see where the PermissionDeniedException has been thrown from.